tuplex/core/include/physical/ResolveTask.h

```diff
@@ -271,7 +275,6 @@ namespace tuplex {
 
         // python output which can't be consolidated, saved as separate list
         void writePythonObject(PyObject* out_row);
```

Comment:
add to docstring maybe: this writes to fallback sink (is that correct?), maybe turn into doxygen string